crates/uv-cli/src/lib.rs

```diff
@@ -2548,6 +2548,13 @@ pub struct RunArgs {
     #[arg(long, conflicts_with = "locked")]
     pub frozen: bool,
 
+    /// Run the given path as a Python script.
+    ///
+    /// Using `--script` will attempt to parse the path as a PEP 723 script,
+    /// irrespective of its extension.
+    #[arg(long)]
```

Comment:
Not sure what the best approach to arg parsing is here — we could explicitly enumerate conflicts (such as ```--extra``` and ```all-extras```), but we don't even warn for these as is when running a script (implicitly) so not sure if it's worth doing. Happy to either way!